doc/version_history.rst

```diff
@@ -6,6 +6,24 @@
 Version History
 ###############
 
+v0.22.0
+-------
+
+Changes:
+
+* Remove support for the sync_pattern field in low-level commands.
+  This requires ts_hexrotcomm 0.28.
+
+Requires:
+
+* ts_rotator_controller 1.3.0
```

Comment:
The version of `ts_rotator_controller` is 1.3.5 now already. Please consider to use a higher version number such as 1.4.0:
https://github.com/lsst-ts/ts_rotator_controller/blob/develop/doc/versionHistory.md

Thanks!

Reply:
Great suggestion.